wordpress-mobile/gutenberg-mobile
# Fix for history that's not empty on a fresh start of the editor

This PR fixes #887 

All the changes are in the GB side companion PR https://github.com/WordPress/gutenberg/pull/15055

**Test 1**
- Start the demo app
- Tap on a RichText block
- Check the Undo/Redo button are not available

**Test 2**
- Start the demo app
- Tap on a RichText block
- Check the Undo/Redo button are not available
- Go to the end of the block
- Tap enter to create a new para
- Tap Undo twice* and the new block should be gone

* => there is another problem with history that I will tackle in another PR. Shortly, Aztec can modify the HTML tags order, so pressing enter on a block does result in sending back to the store an HTML content that could be slightly different from the one set at the beginning, resulting in a new history entry**
